vm.go

```diff
@@ -0,0 +1,60 @@
+/// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+const procPath = "/proc"
+
+var errFound = errors.New("found")
+
+var hypervisors = []string{"qemu"}
+
+// IsVMRunning looks in /proc for a hypervisor process that contains
+// the containerID in its command line
+func IsVMRunning(containerID string) bool {
+	err := filepath.Walk(procPath, func(path string, info os.FileInfo, err error) error {
+		if !info.IsDir() {
+			return filepath.SkipDir
+		}
+
+		cmdlineFile := filepath.Join(path, "cmdline")
+		if _, err := os.Stat(cmdlineFile); os.IsNotExist(err) {
```

Comment:
I don't think the `Stat()` is required as `ReadFile()` will handle `ENOENT` itself.